metrics/testdata/prometheus_metrics

```diff
@@ -376,24 +376,3 @@ container_threads_max{container_env_foo_env="prod",container_label_foo_label="ba
 # HELP container_ulimits_soft Soft ulimit values for the container root process. Unlimited if -1, except priority and nice
 # TYPE container_ulimits_soft gauge
 container_ulimits_soft{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",ulimit="max_open_files",zone_name="hello"} 16384 1395066363000
-# HELP process_cpu_seconds_total Total user and system CPU time spent in seconds.
```

Comment:
Is there a way we can continue to test these?  I'm not sure if it is a good idea to drop coverage here

Reply:
`client_golang` library must be adding them at later stage, I think: https://github.com/prometheus/client_golang/blob/040c98c8e638eb358dfc1b4b04b385a8df21e6a7/prometheus/process_collector.go

Reply:
Ah, I confused it with the container processes metric...